src/fetchers/BitIndexer.ts

```diff
@@ -62,11 +62,13 @@ export class BitIndexer {
   /**
    * Get all accounts whose owner is given key.
    * @param keyInfo
+   * @param role
    */
-  accountList (keyInfo: KeyInfo): Promise<string[]> {
+  accountList (keyInfo: KeyInfo, role?: 'manager'|'owner'): Promise<string[]> {
```

Comment:
As 'owner' is the default `role`, so it would be better to use default assignment: `role: 'manager'|'owner' = 'owner'`